lesson_15/romb_figure.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+# Configure the logger to write to a file
+logging.basicConfig(filename='romb_figure.log', level=logging.INFO)
+
+class Rhombus:
+    def __init__(self, side_a, angle_a):
+        self.logger = logging.getLogger()  # Get the root logger
+        if side_a <= 0:
```

Comment:
Your implementation is not fully correct. The main purpose is to assign classic object attributes in constructor method. Then define additional __setattr__ method for implementation of the logic with task. Please review task another one time and fix your solution. Thanks